package.json

```diff
@@ -17,7 +17,7 @@
     "sass": "1.59.3",
     "sharp": "0.31.3",
     "stripe": "11.15.0",
-    "styled-components": "5.3.9"
+    "styled-components": "^5.3.9"
```

Comment:
Please don't add the caret symbol in the version, so we can make sure that we all are installing the same version.

Reply:
I've fixed on merge